### PR TITLE
[hi] added firewall in hindi

### LINKED ISSUE #764: [hi] Translate `Firewall` into `Hindi`
localization of `Firewall` into Hindi 
- https://glossary.cncf.io/firewall/

### PR DESCRIPTION
**Pull request by starlightknown**
fixes #764

### CODE REVIEW

File: content/hi/firewall.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: Firewall (फ़ायरवॉल)
```
Comment (Garima-Negi) [inline content/hi/firewall.md:2]: We should have the Hindi transliteration first and the english term in parenthesis () here. e.g. like in [https://glossary.cncf.io/hi/continuous_integration/](https://glossary.cncf.io/hi/continuous_integration/)

File: content/hi/firewall.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+---
+
+### यह क्या है
+फ़ायरवॉल (Firewall) एक सिस्टम है जो निर्दिष्ट नियमों के आधार पर नेटवर्क ट्रैफ़िक को फ़िल्टर करता है। फायरवॉल हार्डवेयर, सॉफ्टवेयर या दोनों का संयोजन हो सकता है।
+
+### समस्या 
+डिफ़ॉल्ट रूप से, जब तक वे नेटवर्क के रूटिंग नियमों का पालन करते हैं, नेटवर्क किसी को भी प्रवेश करने और प्रस्थान करने की अनुमति देगा। इस डिफ़ॉल्ट व्यवहार के कारण, नेटवर्क को सुरक्षित करना चुनौतीपूर्ण है। उदाहरण के लिए, एक माइक्रोसर्विस-आधारित बैंकिंग ऐप में, सेवाएं अपने नेटवर्क के माध्यम से अत्यधिक संवेदनशील वित्तीय डेटा संचारित करके एक दूसरे के साथ संचार करती हैं। एक दुर्भावनापूर्ण अभिनेता नेटवर्क में घुसपैठ कर सकता है, संचार को बाधित कर सकता है, और अगर जगह में कोई फ़ायरवॉल नहीं है तो नुकसान कर सकता है।
+
+### समाधान 
+फ़ायरवॉल पूर्व-निर्धारित नियमों का उपयोग करके नेटवर्क ट्रैफ़िक की जाँच करता है। सभी ट्रैफ़िक को फ़िल्टर किया जाता है, और अविश्वसनीय या संदिग्ध स्रोतों से आने वाले किसी भी ट्रैफ़िक को अवरुद्ध कर दिया जाता है - केवल स्वीकार किए जाने के लिए कॉन्फ़िगर किया गया ट्रैफ़िक ही अंदर आता है। फ़ायरवॉल सुरक्षित और नियंत्रित आंतरिक विश्वसनीय नेटवर्क के बीच एक अवरोध स्थापित करते हैं।
```
[the remark this answers is not in the document]
Comment (Garima-Negi) [reply content/hi/firewall.md:15]: just a suggestion that filter can be further explained as `छानकर शुद्ध करना` .

**Review by Garima-Negi — CHANGES_REQUESTED** [review]

File: content/hi/firewall.md (hunk 1 above)
Comment (starlightknown) [reply content/hi/firewall.md]: done

**Review by Garima-Negi — APPROVED** [review]

**Review by jayesh-srivastava — APPROVED** [review]
lgtm

**Comment by starlightknown** [comment]
done

**Comment by jayesh-srivastava** [comment]
@starlightknown Seems like other commits have been pushed to the branch. PTAL.

**Comment by starlightknown** [comment]
yeah, how can I revert it?

**Comment by Garima-Negi** [comment]
Hi @starlightknown , your change to merge into `cncf:dev-hi` is correct.
However, `starlightknown:main`  tells me that you made firewall.md related changes in your local machines `main` branch. 
if possible, can you pull latest main and then create a new branch for firewall.md ?

**Comment by starlightknown** [comment]
> Hi @starlightknown , your change to merge into `cncf:dev-hi` is correct. However, `starlightknown:main` tells me that you made firewall.md related changes in your local machines `main` branch. if possible, can you pull latest main and then create a new branch for firewall.md ?



> Hi @starlightknown , your change to merge into `cncf:dev-hi` is correct. However, `starlightknown:main` tells me that you made firewall.md related changes in your local machines `main` branch. if possible, can you pull latest main and then create a new branch for firewall.md ?

done #1147

**Pull request closed without merging**